### DISCUSSION TITLE
Yaade tool - productive release date of the fix available in the beta version

### DISCUSSION THREAD

**Discussion by ndbg2024** [community/community#147833] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The issue, that we can not sync the groups from EntraID is fixed in the beta version called nightly. 
Here's the link for the same:
https://hub.docker.com/r/esperotech/yaade/tags

Want to know the productive release date for this fix.
Could someone help us in this regards.

**Comment by mecodeatlas** [comment]
Hi @ndbg2024,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
